config.ini.sample

```diff
@@ -60,3 +60,163 @@ days_of_data = 2
 
 [metric_reporter]
 reports_dir = reports
+
+[google_sheet_uploader]
+service_account_file = google_sheets_key.json
+mappings = [
```

Comment:
I wonder if there is a structure we could use so we don't need to repeat the sheet_id?